### ISSUE TITLE
Can i know which thumb value has been changed

### ISSUE THREAD

**Issue by bhuvneshvarma** [anothem/android-range-seek-bar#16]
Inside setOnRangeSeekBarChangeListener  can i know which thumb value has been changed?Is there any method which let user know which thumb has been moved?

**Comment by anothem** [comment]
No, you just get the new values, so unless you keep track yourself of the previous values, there's no way to do this.
